tests/select_for_update/tests.py

```diff
@@ -86,30 +96,56 @@ def test_nowait_raises_error_on_block(self):
         """
         self.start_blocking_transaction()
         status = []
-
```

Comment:
I'd rather if you made these changes including rewriting existing docstrings in a separate commit.